### PR TITLE
Cleanup coordinate encoder unit tests

### LINKED ISSUE #2043: Cleanup coordinate encoder unit tests
Part of #2000

### PR DESCRIPTION
**Pull request by oxtopus**
Fixes #2043

### CODE REVIEW

**Comment by david-ragazzi** [comment]
:+1

File: tests/unit/nupic/encoders/coordinate_test.py [hunk 1]
```diff
@@ -255,15 +251,8 @@ def testEncodeAdjacentPositions(self, verbose=False):
                                           num=1)
       allOverlaps[i] = overlaps[0]
 
-    self.assertTrue(np.min(allOverlaps) > minThreshold)
-    self.assertTrue(np.average(allOverlaps) > avgThreshold)
-
-    if verbose:
```
Comment (chetan51) [inline tests/unit/nupic/encoders/coordinate_test.py]: Why remove this? It was helpful in debugging.

File: tests/unit/nupic/encoders/coordinate_test.py [hunk 2]
```diff
@@ -340,15 +325,6 @@ def overlapsForRelativeAreas(n, w,
     outputB = encode(encoder, newPosition, newRadius)
     overlaps[i] = overlap(outputA, outputB)
 
-  if verbose:
```
Comment (chetan51) [inline tests/unit/nupic/encoders/coordinate_test.py]: Why remove this? It was helpful in debugging.

**Comment by oxtopus** [comment]
@chetan51 I restored the verbose output

**Comment by david-ragazzi** [comment]
After Travis is green we merge it..

**Comment by chetan51** [comment]
:+1:

**Pull request merged**